doc/nrf/index.rst

```diff
@@ -3,6 +3,23 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v2.1.99-dev2 is a development tag and will be replaced by v2.2.0 in the future.
+
+   The development tag contains the following updates:
+
+   * New :ref:`wifi_scan_sample` sample.
+   * New :ref:`wifi_station_sample` sample.
+   * New :ref:`wifi_ficr_prog` sample, adding support for FICR/OTP programming.
```

Comment:
I think here we need to list the radio test sample. This was not part of any previous release. the "FICR" Sample is only a part of this sample that support commands for FICR, OTP programming.